Sustainsys.Saml2.AspNetCore2/Saml2Options.cs

```diff
@@ -20,6 +21,8 @@ public Saml2Options()
             {
                 ModulePath = "/Saml2"
             };
+            // uses the headers directly when no chunk size is set
+            CookieManager = new ChunkingCookieManager();
```

Comment:
The default value can be set in PostConfigureOptions instead, to avoid creating an object that is then over-written by custom config:
https://github.com/Sustainsys/Saml2/blob/0859e67392ea613fcb82a8d7e0e9a6c1c4de6ab6/Sustainsys.Saml2.AspNetCore2/PostConfigureSaml2Options.cs#L31